pkg/sti/build.go

```diff
@@ -150,9 +150,16 @@ func (h *buildHandler) PostExecute(containerID string, location string) error {
 		}
 	}
 
-	cmd := []string{}
+	image, err := h.docker.GetBuilderImage(containerID)
```

Comment:
There's no need to reach container, just invoke it with the name of the builder image:
```
image, err := h.docker.GetBuilderImage(h.request.BaseImage)
```
and the `GetBuilderImage` method should just access the `InspectImage` method. There's no need to add new ones.